src/server/lib/log.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * An alias for console.dir.
+ *
+ * This function is a no-op if process.env.VERBOSE is not set.
+ *
+ * @param {...*} messages
+ */
+let dir = (...messages) => {}
+if (process.env.VERBOSE) {
```

Comment:
rather than looking in `process.env` directly, can this setting live in `server/environment.js` ?

Reply:
Oh yeah.  Good idea!